sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/LoadBalancerOutboundRule.java

```diff
@@ -0,0 +1,226 @@
+    int idleTimeoutInMinutes();
+
+    /** @return if TCP reset is enabled */
+    boolean tcpResetEnabled();
+
+    /** @return outbound rule protocol */
+    LoadBalancerOutboundRuleProtocol protocol();
+
+    /** Grouping of outbound rule definition stages. */
+    interface DefinitionStages {
+        /**
+         * The first stage of the outbound rule definition.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface Blank<ParentT> extends LoadBalancerOutboundRule.DefinitionStages.WithProtocol<ParentT> {
+        }
+
+        /**
+         * The final stage of the outbound rule definition.
+         *
+         * <p>At this stage, any remaining optional settings can be specified, or the outbound rule definition can be
+         * attached to the parent load balancer definition.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface WithAttach<ParentT>
+            extends Attachable.InDefinition<ParentT>,
+            LoadBalancerOutboundRule.DefinitionStages.WithEnableTcpReset<ParentT>,
+            LoadBalancerOutboundRule.DefinitionStages.WithIdleTimeout<ParentT> {
+        }
+
+        /**
+         * The stage of an outbound rule definition allowing to specify the transport protocol.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface WithProtocol<ParentT>
+            extends HasProtocol.DefinitionStages.WithProtocol<LoadBalancerOutboundRule.DefinitionStages.WithBackend<ParentT>, LoadBalancerOutboundRuleProtocol> {
+        }
+
+        /**
+         * The stage of an outbound rule definition allowing to specify a backend pool for the rule to apply to.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface WithBackend<ParentT>  {
+            LoadBalancerOutboundRule.DefinitionStages.WithFrontend<ParentT> fromBackend(String name);
+        }
+
+        /**
+         * The stage of an outbound rule definition allowing to specify the frontend IP address.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface WithFrontend<ParentT> {
+            LoadBalancerOutboundRule.DefinitionStages.WithAttach<ParentT> toFrontend(String name);
+        }
+
+        /**
+         * The stage of an outbound rule definition allowing to specify the frontend IP address.
+         *
+         * @param <ParentT> the stage of the parent definition to return to after attaching this definition
+         */
+        interface WithFrontends<ParentT> {
+            LoadBalancerOutboundRule.DefinitionStages.WithAttach<ParentT> toFrontends(List<String> names);
+        }
```

Comment:
Seems no step to reach this interface?

May it be merged with `WithFrontend`?

Reply:
updated